### PR TITLE
[feat][case&pat] add-button, scss scoping

### PR DESCRIPTION
**Pull request by wormwithnoname**
**What this PR does:**

- added "add-button"
- scss styles scoping and modifications

**Screenshots:**
image
image

### CODE REVIEW

File: src/components/LoginCard/index.jsx [hunk 1]
```diff
@@ -104,7 +104,7 @@ function LoginCard() {
             </Button>
             <Text className="Login-subtitle">
               Don&apos;t have an account yet?{' '}
-              <Link className="Login-subtitle" to="./signup">
+              <Link className="Login-link" to="./signup">
```
Comment (itskichaem) [inline src/components/LoginCard/index.jsx]: let use signup route constant

File: src/components/SignupCard/index.jsx [hunk 2]
```diff
@@ -124,7 +124,7 @@ function SignupCard() {
         <Space direction="vertical" size="medium">
           <Text className="Login-subtitle">
             Have an account?{' '}
-            <Link className="Login-subtitle" to="./login">
+            <Link className="Login-link" to="./login">
```
Comment (itskichaem) [inline src/components/SignupCard/index.jsx]: let use login route constant

**Review by itskichaem — CHANGES_REQUESTED** [review]

File: src/components/LoginCard/index.jsx (hunk 1 above)
Comment (wormwithnoname) [reply src/components/LoginCard/index.jsx]: resolved kuya!

File: src/components/SignupCard/index.jsx (hunk 2 above)
Comment (wormwithnoname) [reply src/components/SignupCard/index.jsx]: resolved :>

**Review by itskichaem — APPROVED** [review]

**Pull request merged**